docs/reference/execution-providers/CUDA-ExecutionProvider.md

```diff
@@ -0,0 +1,80 @@
+---
+title: CUDA
+parent: Execution Providers
+grand_parent: Reference
+nav_order: 1
+---
+
+# CUDA Execution Provider
+
+The CUDA Execution Provider enables hardware accelerated computation on Nvidia CUDA-enabled GPUs.
+
+## Build
+For build instructions, please see the [BUILD page](../../how-to/build.md#CUDA).
+
+## Configuration Options
+The CUDA Execution Provider supports the following configuration options.
+
+### device_id
+The device ID.
+
+### cuda_mem_limit
+The size limit of the device memory arena in bytes. This size limit is only for the execution provider's arena. The total device memory usage may be higher.
```

Comment:
Sorry for adding another comment. Maybe it is worth specifying the "default" values for each of these parameters ? For example, reading the doc, I understand what the valid values for `cudnn_conv_algo_search` are and what each option means. What is hard to parse is the "default" value for each.